### PR TITLE
InvalidID-FAQ-Update.md

### PR DESCRIPTION
**Pull request by ptolleyedc**

### CODE REVIEW

**Comment by ptolleyedc** [comment]
I have not tested this locally, Nathan is taking a look at doing that. This is to move forward to see how the page builds so we are not entirely blocked

File: _pages/faqs.md [hunk 1]
```diff
@@ -72,20 +72,52 @@ The systems and methods for remote identity proofing require an ID; without one
...
+    <b style="color: #000000; font-size: 20px; font-weight: bold;">🚫 The following IDs are not eligible 🚫</b><br><br>
+    <div style="text-align: left;">
+        • Passports, Military IDs, Green cards, or other federal government identity cards<br>
+        • Identification cards issued by a city or municipality<br>
+        • Library cards, school or college IDs, workplace badges, etc.<br>
+        • Temporary IDs<br>
+        • Paper copies of your identification<br>
+        • Digital pictures of your ID displayed on a screen<br><br>
+    </div>
+    <img src="https://feedback.gsa.gov/CP/Graphic.php?IM=IM_ddjGgMJHuJ5smxM" style="width: 65%;"><br><br>
```
Comment (astrogeco) [inline _pages/faqs.md:119]: Upload picture directly to repository and replace `src`

Add empty `alt` tag (see home page for example)

File: _pages/faqs.md [hunk 2]
```diff
@@ -72,20 +72,52 @@ The systems and methods for remote identity proofing require an ID; without one
...
+    @media screen and (max-width: 600px) {
+      .editable-box, .editable-box.grey {
+        width: 100%; /* Adjust width for smaller screens */
+      }
+</style>
+
+<div class="editable-box">
+    <b style="color: #fbfdff; font-size: 20px; font-weight: bold;">✅ The following IDs are eligible ✅</b><br><br>
+   <span style="color: #fbfdff; font-weight: bold;">Driver’s licenses and state-issued ID card</span> <span style="color: #fbfdff;">from the 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, Mariana Islands and Puerto Rico)</span><br><br>
+    <img src="https://feedback.gsa.gov/CP/Graphic.php?IM=IM_cM91zAksWWMyOdU"><br><br>
```
Comment (astrogeco) [inline _pages/faqs.md:106]: Upload picture directly to repository and replace `src`

Add empty `alt` tag

**Review by astrogeco — CHANGES_REQUESTED** [review]

File: _pages/faqs.md [hunk 3]
```diff
@@ -72,20 +72,52 @@ The systems and methods for remote identity proofing require an ID; without one
 
 ### Which IDs are eligible?
 
-U.S. state and territory-issued identification cards with a profile photo are eligible, including:
-- Driver’s licenses from the 50 states, the District of Columbia (DC), and other US territories (Guam, US Virgin Islands, Mariana Islands and Puerto Rico)
-- Non-driver’s license state-issued ID card: This is an identity document issued by the state, the District of Columbia (DC), or US territory that asserts identity but does not give driving privileges.
-
-**Please use the same ID for all pictures.** Note that a worn or damaged picture may prevent the identity verification process from working.
```
Comment (astrogeco) [inline _pages/faqs.md:79]: Need to include this statement somewhere

**Review by astrogeco — CHANGES_REQUESTED** [review]

**Review by astrogeco — APPROVED** [review]

**Pull request merged**